config/locales/en.yml

```diff
@@ -1,11 +1,22 @@
 en:
   coronavirus:
     pages:
+      error:
```

Comment:
Sorry another comment on key organisation. 

This seems a touch arbitrary to have just one section of error within coronavirus.pages when so much is in there - and there's probably a few other errors in there. Perhaps it could be under the action it's defined under `update` or alternative idea would be putting all flash messages for the controller under one header of `flashes`

Reply:
This is the only error that applies to the whole controller as it's triggered by a before action. All of the other errors are action specific so I'm inclined to leave this where it is.

Reply:
Sure, I guess I find it confusing just due to how broad the key of coronavirus.pages is, given that spans multiple controllers.